src/main/java/org/icgc/argo/program_service/properties/SpringValidatorConfig.java

```diff
@@ -0,0 +1,28 @@
+package org.icgc.argo.program_service.properties;
+
+import org.springframework.boot.autoconfigure.orm.jpa.HibernatePropertiesCustomizer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Lazy;
+
+import javax.validation.Validator;
+import javax.validation.ValidatorFactory;
+import java.util.Map;
+
+@Configuration
+@Lazy
+class SpringValidatorConfiguration {
```

Comment:
Add a comment explaining why this needs to be defined
https://stackoverflow.com/questions/50212117/spring-boot-hibernate-custom-constraint-doesnt-inject-service/50213178#50213178
https://stackoverflow.com/questions/2712345/jsr-303-dependency-injection-and-hibernate